src/app/(proper_react)/(redesign)/(authenticated)/admin/emails/actions.tsx

```diff
@@ -123,15 +124,16 @@ export async function triggerMonthlyActivity(emailAddress: string) {
 
 export async function triggerFirstDataBrokerRemovalFixed(emailAddress: string) {
   const l10n = getL10n();
+  const randomScanResult = createRandomScanResult({ status: "removed" });
 
   await send(
     emailAddress,
     l10n.getString("email-first-broker-removal-fixed-subject"),
     <FirstDataBrokerRemovalFixed
       data={{
-        dataBrokerName: "Data broker name",
-        dataBrokerLink: process.env.SERVER_URL ?? "",
-        removalDate: new Date(Date.now()),
+        dataBrokerName: randomScanResult.data_broker,
```

Comment:
Hm, is this intentional or left over from testing?

Reply:
This is intentional to closer match the design specs for testing. This is only the email we send out from the admin email panel.